ROADMAP.md

```diff
@@ -0,0 +1,71 @@
+# Roadmap
+
+> This document describes what additional tools/configs could be added to this monorepo.
+
+## New packages to add
+
+### ESLint
+
+We should be able to lint our javascript projects through ESLint and its configuration.
+
+#### Base
+
+Base configuration which should be used everywhere.
+
+#### Node
+
+Specific configuration for Node.js backends.
+
+#### React
+
+Specific configuration for React apps.
+
+#### React Native
+
+Specific configuration for React Native apps.
+
+#### Typescript
+
+Specific configuration for apps using Typescript.
+
+#### Jest
+
+Specific configuration for Jest testing.
+
+> [ESLint Plugin Jest][eslint-plugin-jest] is great candidate.
+
+#### GraphQL
+
+Specific configuration for apps using GraphQL.
+
+> [ESLint Plugin GraphQL][eslint-plugin-graphql] is great candidate.
+> [GraphQL ESLint plugin][graphql-eslint-plugin]
+
+### Markdown
+
+We should be able lint and format our readmes and other markdown documentation.
+
+> [Remark CLI][remark-cli] is great candidate.
```

Comment:
If linting is all we need, I have quite a good experience with https://github.com/DavidAnson/markdownlint-cli2 (compatible with MDX, to certain extent).